site/_includes/layouts/extension-samples.njk

```diff
@@ -0,0 +1,172 @@
+{% extends "layouts/base.njk" %}
```

Comment:
I have placed it in the layout because there is no specific language content in the njk template (localization-related content has been moved to the i18n file). Here it functions similarly to `site/en/docs/extensions/index.md`, which contains English titles and descriptions.

Reply:
@matthiasrohmer, any thoughts on this?

Reply:
I can follow the train of thought, but following the fugu-showcase example here would be good. So, putting the contents of this file directly in `site/en/docs/extensions/samples/index.md`(and renaming it to njk`) - it's unlikely we'll have localized versions of this page (which is when the extraction would make sense) and it's more intuitive to do changes for new starters as they would find the code they expect inside the page rather than having to further search for the used template.

Reply:
Done. https://github.com/GoogleChrome/developer.chrome.com/pull/7029/commits/7a20eba71ac351c2145b117c3d326d90f5cdf882